framework/platform/bpl/include/bpl/bpl_uci.h

```diff
@@ -0,0 +1,63 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _BPL_UCI_H_
+#define _BPL_UCI_H_
+
+#include "bpl.h"
+#include "bpl_err.h"
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace beerocks {
+namespace bpl {
+
+/****************************************************************************/
+/******************************* Definitions ********************************/
+/****************************************************************************/
+
+/****************************************************************************/
+/******************************* Structures *********************************/
+/****************************************************************************/
+
+/****************************************************************************/
+/******************************** Functions *********************************/
+/****************************************************************************/
+
+bool uci_entry_exists(const std::string &config_file, const std::string &entry_type,
```

Comment:
Currently converting it to a class causes too many design issues with regard to whom would be responsible with handling the instance,
This will be refactored as part of phase 2

Reply:
https://jira.prplfoundation.org/browse/PPM-315